src/python/pants/backend/python/lint/black/rules_integration_test.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
-
+from dataclasses import replace
```

Comment:
Nit: usually we do indeed use `from` imports. But with `replace`, I recommend instead importing `dataclasses` directly and using `dataclasses.replace` because `replace()` is a really generic name, and plausibly could mislead the reader to think it's a builtin similar to `copy`.